frontend/src/components/Alerts/AlertsBanner.tsx

```diff
@@ -21,15 +21,27 @@ const Center = styled.div`
     align-items: center;
 `
 
+export enum AlertCategory {
+    ERROR,
+    WARNING,
+    SUCCESS,
+}
+
 interface AlertProps {
     children: ReactNode
     dismissAlert: () => void
+    alertCategory: AlertCategory
 }
 
-export function AlertBanner({ children, dismissAlert }: AlertProps) {
+export function AlertBanner({ children, dismissAlert, alertCategory }: AlertProps) {
```

Comment:
File is called AlertsBanner while function is called AlertBanner. Suggest renaming file to AlertBanner